ml-proto/runtests.py

```diff
@@ -32,29 +37,48 @@ def _compareLog(self, fileName, logPath):
     except IOError:
       pass
 
+  def _readHeaders(self, fileName):
+    expected_exit_code = 0
```

Comment:
I have always been told that python was strongly in favor of snake_casing? Staying consistent with the original style, though.